test/runtests.jl

```diff
@@ -2,36 +2,38 @@ using Test
 using PSDMatrices
 using LinearAlgebra
 
-M1 = [1 1; 2 20]
-M2 = [1.0 1.0; 2.0 20.0]
-M3 = big.([1.0 1.0; 2.0 20.0])
+Mbase = [1 1; 2 20]
+eltypes = (Int64, Float64, BigFloat)
 
-@testset "$M" for M in (M1, M2, M3)
-    S = PSDMatrix(M)
+@testset "PSDMatrices.jl" begin
+    @testset "eltype=$t" for t in eltypes
+        M = t.(Mbase)
+        S = PSDMatrix(M)
 
-    @test eltype(S) == eltype(M)
+        @test eltype(S) == eltype(M) == t
 
-    @testset "My exports" begin
-        @test norm(todense(S) - M * M') == 0.0
-        @test todense(X_A_Xt(S, M)) ≈ M * todense(S) * M'
-        @test choleskify_factor(S).L ≈ cholesky(todense(S)).U'
-        @test todense(add_cholesky(S, S)) ≈ todense(S) + todense(S)
-        @test todense(add_qr(S, S)) ≈ todense(S) + todense(S)
-    end
+        @testset "My exports" begin
```

Comment:
what about those new indentations? Is it on purpose?

Reply:
It''s because I wrapped the whole file in a big testset. I personally prefer that and find it convenient to run all the tests, even if some fail.

Reply:
ah I missed that you added the outer test set. My bad.